api/src/main/java/net/kyori/adventure/text/AbstractComponent.java

```diff
@@ -44,6 +45,8 @@
 @Debug.Renderer(text = "this.debuggerString()", childrenArray = "this.children().toArray()", hasChildren = "!this.children().isEmpty()")
 @Deprecated
 public abstract class AbstractComponent implements Component {
+  private static final HashMap<Integer, Component> COMPONENT_POOL = new HashMap<>();
```

Comment:
The diamond operator here is just a decoration. You should remove it.

Reply:
I think the Diamond operator wants friends instead…



```suggestion
  private static final HashMap<Integer, Component> COMPONENT_POOL = new HashMap<Integer, Component>();
```

Reply:
But functionally that still *does* nothing. It just adds more clutter to the code!